genny/fizz/ctable/create_table_test.go

```diff
@@ -0,0 +1,32 @@
+package ctable
+
+import (
+	"testing"
+
+	"github.com/gobuffalo/genny/gentest"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_New(t *testing.T) {
+	r := require.New(t)
+
+	g, err := New(&Options{
+		TableName: "widgets",
+		Name:      "create_widgets.fizz",
+	})
+	r.NoError(err)
+
+	run := gentest.NewRunner()
+	run.With(g)
+
+	r.NoError(run.Run())
+
+	res := run.Results()
+
+	r.Len(res.Commands, 0)
+	r.Len(res.Files, 1)
+
+	f := res.Files[0]
+	r.Equal("create_widgets.fizz", f.Name())
+	r.Equal("", f.String())
```

Comment:
why is this expecting an empty string? default is table with just timestamps isn't it?